src/test/kotlin/com/hibob/academy/dao/PetRecordDaoTest.kt

```diff
@@ -135,4 +135,30 @@ class PetRecordDaoTest @Autowired constructor(private val sql: DSLContext)  {
         // Optional: Check that there are no unexpected pet types in the map
         assertTrue(petsMap.keys.containsAll(listOf(PetType.DOG, PetType.CAT)), "Test failed: Map contains unexpected pet types!")
     }
+
+    @Test
+    fun `Test adopt Multiple pets`() {
+        val id1 = dao.insertNewPet(PetRecord("Waffle", PetType.DOG, LocalDate.now(), companyId, null))
+        val id2 = dao.insertNewPet(PetRecord("Rex", PetType.DOG, LocalDate.now(), companyId, null))
+        val id3 = dao.insertNewPet(PetRecord("George", PetType.DOG, LocalDate.now(), companyId, -5L))
```

Comment:
Why do you use negative numbers as ids?
If you do enforce that - it will work - but usually positive numbers are used as ids

Reply:
You have other negative numbers there - please review your whole file for those